content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -20,73 +23,96 @@ Generate leads
+(:menuselection:`CRM app --> Sales --> My Pipeline`).
+
+Lead mining requests are also available through :menuselection:`CRM app --> Configuration --> Lead
+Mining Requests`, or through :menuselection:`CRM app --> Leads --> Leads`, where the
+:guilabel:`Generate Leads` button is also available.
 
 .. image:: lead_mining/generate-leads-button.png
    :align: center
    :alt: The Generate Leads button to use the lead mining feature.
 
-Click on the :guilabel:`Generate Leads` button, and a window will appear offering a variety of
+Click the :guilabel:`Generate Leads` button, and a pop-up window appears, offering a variety of
 criteria by which to generate leads.
 
 .. image:: lead_mining/generate-leads-popup.png
    :align: center
    :alt: The pop-up window with the selection criteria in order to generate leads in Odoo.
 
 Choose to generate leads for :guilabel:`Companies` to get company information only, or choose
-:guilabel:`Companies and their Contacts` to get company information as well as individual employee
-contact information. When targeting :guilabel:`Companies and their Contacts`, there is an option to
-filter contacts based on :guilabel:`Role` or :guilabel:`Seniority`.
+|CC| to get company information and individual employee contact information.
 
-Additional filtering options include:
-
-- :guilabel:`Size`: filter leads based on the number of employees at the company
-- :guilabel:`Countries`: filter leads based on the country (or countries) they are located in
-- :guilabel:`States`: further filter leads based on the state they are located in, if applicable
-- :guilabel:`Industries`: filter leads based on the specific industry they work in
-- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to
-- :guilabel:`Salesperson`: choose which person(s) on the Sales Team the leads will be assigned to
-- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found
+.. note::
+   When targeting |CC|, additional options are available to filter contacts based on
+   :guilabel:`Role` or :guilabel:`Seniority`.
+
+Additional filtering options include the following:
+
+- :guilabel:`Countries`: filter leads based on the country (or countries) they are located in.
+- :guilabel:`States`: further filter leads based on the state in which they are located, if
+  applicable.
+- :guilabel:`Industries`: filter leads based on the specific industry they work in.
+- :guilabel:`Filter on Size`: tick this checkbox to specify the number of employees at the company.
+  This generates a field labeled :guilabel:`Size`. Fill in the blanks to create a range for the
+  desired company size.
+- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to.
+- :guilabel:`Salesperson`: choose which member of the Sales Team the leads will be assigned to.
+- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found.
 
 .. note::
```

Comment:
Given how important this is for the EU (and Odoo as whole), I'd change the block accordingly:
```suggestion
.. important::
```